RecoEgamma/EgammaElectronProducers/plugins/TrackingRegionsFromSuperClustersProducer.h

```diff
@@ -181,6 +195,11 @@ fillDescriptions(edm::ConfigurationDescriptions& descriptions)
   desc.add<double>("deltaPhiRegion",0.4);
   desc.add<double>("deltaEtaRegion",0.1);
   desc.add<bool>("useZInVertex", false);
+  desc.add<bool>("useZInBeamspot", true);
+  desc.add<bool>("useDefaultZ", false);
```

Comment:
As already said, I would remove this parameter as it is both useless and confusing